package.json

```diff
@@ -283,15 +283,16 @@
     "url-loader": "2.2.0",
     "webpack": "4.41.2",
     "webpack-bundle-analyzer": "3.6.0",
-    "webpack-cli": "3.3.9",
-    "webpack-dev-server": "3.9.0",
-    "webpack-merge": "4.2.2"
+    "webpack-cli": "3.3.9",    "webpack-dev-server": "3.9.0",
+    "webpack-merge": "4.2.2",
+    "nock": "10.0.6"
   },
   "dependencies": {
     "@hot-loader/react-dom": "16.10.2",
     "@rebass/forms": "4.0.6",
     "@styled-system/theme-get": "5.1.2",
     "axios": "0.19.0",
+    "bech32": "^1.1.3",
```

Comment:
This dependency should be pinned as an exact match (using `yarn add bech32 --exact`)